src/state/atoms/releaseAtom.ts

```
@@ -58,16 +59,21 @@ export const hidePreviewBannerAtom = atomWithToggle(initialHidePreviewBanner, as
 
 /**
  * Atom for layouts to signal that the preview banner should be hidden.
- * Used by the Lightwell layout to hide the BetaSwitcher directly,
- * instead of checking the URL path.
+ * Initialized from the current pathname so the banner never renders on Lightwell routes,
+ * even before any component mounts.
  */
-export const layoutBannerHiddenAtom = atom(false);
+export const layoutBannerHiddenAtom = atom(
+  window.location.pathname === LIGHTWELL_PATH || window.location.pathname.startsWith(`${LIGHTWELL_PATH}/`)
+);
```

ESLint

[error] Replace `⏎··window.location.pathname·===·LIGHTWELL_PATH·||·window.location.pathname.startsWith(`${LIGHTWELL_PATH}/`)⏎` with `window.location.pathname·===·LIGHTWELL_PATH·||·window.location.pathname.startsWith(`${LIGHTWELL_PATH}/`)`

(prettier/prettier)